baus/summaries/affordable_housing_summaries.py

```diff
@@ -119,6 +119,11 @@ def deed_restricted_units_growth_summary(year, initial_summary_year, final_year,
             # growth in units
             dr_growth[col+'_growth'] = dr_growth[col+"_"+str(final_year)] - dr_growth[col+"_"+str(initial_summary_year)]
 
+            # percent geography's growth of households/jobs/etc. of all regional growth in households/jobs/etc.
+            dr_growth[col+'_pct_of_regional_growth'] = (round(((dr_growth[col+"_growth"]) / 
+                                                               (dr_growth[col+"_"+str(final_year)].sum() - 
+                                                                dr_growth[col+"_"+str(initial_summary_year)].sum())) * 100, 2))
```

Comment:
Isn't 
`(dr_growth[col+"_"+str(final_year)].sum() - 
                                                                dr_growth[col+"_"+str(initial_summary_year)].sum())`
strictly speaking always just the same as `(dr_growth[col+"_growth"])`, so the statement could simplify to
`(dr_growth[col+"_growth"] / dr_growth[col+"_growth"].sum() ).mul(100).round(2)` ?

Reply:
The denominator is regional growth, which I haven't calculated elsewhere I don't think, but let me know if I'm missing something.

Reply:
Oh sorry got it. Yeah I can change that.